python/cuml/test/test_umap.py

```diff
@@ -71,6 +71,9 @@ def test_umap_fit_transform_score(nrows, n_feats):
     embedding = model.fit_transform(data)
     cuml_embedding = cuml_model.fit_transform(data, convert_dtype=True)
 
+    assert not np.isnan(embedding).any()
```

Comment:
Great idea to look for nans in embedding at different locations. Here, you are testing the original implementation. Is it on purpose?

Reply:
I'd like to test the non-consistent path to better test which configurations appear to result in nan values.